packages/components/package.json

```diff
@@ -47,31 +47,31 @@
     "@porsche-design-system/utilities-v2": "0.0.0",
     "@splidejs/splide": "^4.1.4",
     "@stencil/core": "~2.20.0",
-    "jss": "^10.9.2",
+    "jss": "^10.10.0",
```

Comment:
I noticed that we have `"jss": "^10.9.2",` in `/packages/utilities/projects/styles/package.json` - should we also update it there?

Reply:
See comment in angular